pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -2421,7 +2429,16 @@ protected void handleAddPartitionToTxn(CommandAddPartitionToTxn command) {
                                 ex.getMessage()));
                         transactionMetadataStoreService.handleOpFail(ex, tcId);
                     }
-            }));
+                });
+    }
+
+    private CompletableFuture<Void> failedFutureTxnNotOwned(TxnID txnID) {
+        String msg = String.format(
```

Comment:
why are you using the logger this way ?

Reply:
it's also in the message of the exception